applications/nrf5340_audio/README.rst

```diff
@@ -942,11 +942,11 @@ See the following examples of the parameter usage with the command run from the
 
      python buildprog.py -c app -b debug -d both
 
-* Example 2: The following command builds the application as in *example 1*, but with the DFU internal flash memory layout enabled and using the minimal size of the network core bootloader:
+* Example 2: The following command builds the both core, and with the DFU internal flash memory layout enabled and using the minimal size of the network core bootloader:
```

Comment:
```suggestion
* Example 2: The following command builds both cores, using the DFU internal flash memory and the minimal size of the network core bootloader:
```

Reply:
I thought we had to use external flash if both cores are updated?

Reply:
We should build both core if DFU is enabled either internal or external layout.
Both cores can be updated one by one if internal flash.
Some of our customers might want to experience this feature without flash.

Reply:
Ok